### ISSUE TITLE
firewall checker no audio

### ISSUE THREAD

**Issue by AFAskar** [izer-xyz/3cx-docker#12]
i just used the 3cx-docker container to use 3cx but all ports end in failure i am hosting it on an OCI arm64 server in the cloud and i've forwarded all of the ports in https://www.3cx.com/docs/ports/ except the smtp port is there anyway to fix this issue

**Comment by AFAskar** [comment]
i tried editing the run command to this but it still didn't fix it 
docker run \
>   -d  \
>   -t \
>   --tmpfs /tmp \
>   --tmpfs /run \
>   --tmpfs /run/lock \
>   -v      /sys/fs/cgroup:/sys/fs/cgroup:ro \
>   -p      5015:5015 \
>   -p      5000:5000 \
>   -p      5001:5001 \
>   -p      5060:5060 \
>   -p      5060:5060/udp \
>   -p      5090:5090 \
>   -p      5090:5090/udp \
>   -p      7000:10999/udp \
>           ghcr.io/izer-xyz/3cx:latest

and i apologize if the answer was obvious i am completely new to docker and Linux

**Comment by sinegar** [comment]
It sounds like a firewall issue. Either on the infrastructure (e.g. VPN) or your docker host.

1) Make sure the ports are allowed (I assumed OCI is oracle cloud):

https://cloud.oracle.com/networking/vcns -> Subnet -> Security List Details

2) What linux do you use (e.g. Oracle Linux 8)? Is the firewall on?

**Comment by AFAskar** [comment]
I have oppened all ports related to 3cx in the ports page of the docs in
the security list of my subnet
I use ubuntu 20.04 with ufw where i allowed all the ports being tested

**Comment by sinegar** [comment]
What do you mean by “all ports end in failure”? What is the issue?

**Comment by AFAskar** [comment]
The firewall checker gives a full cone error i’ll screen shot it when i get
home

**Comment by sinegar** [comment]
(just replaced this comment as I misunderstood the issue)

I tested the firewall checker and also the troubleshooting page (https://www.3cx.com/docs/firewall-checker/). I get a bunch of errors due to port mapping, which after the docs (https://www.3cx.com/docs/firewall-checker/#h.ewf2rcjej9o7 - port preservation) may cause issues with some providers.  I never had issues but I only have minimal usage. 

Can you post a screenshot of the firewall checker to see if there is anything obvious?

**Comment by AFAskar** [comment]
this is the output of the firewall checker 
```
resolving 'stun-eu.3cx.com'... done
resolving 'stun2.3cx.com'... done
resolving 'stun3.3cx.com'... done
resolving 'sip-alg-detector.3cx.com'... done
testing 3CX PhoneSystem 01 SIP Server...
stopping service... done
detecting SIP ALG... failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 5060...
testing 3CX PhoneSystem Media Server...
stopping service... done
testing port 5090... done
testing ports [9000..9398]...
testing port 9000... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9002... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9004... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9006... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9008... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9010... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9012... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9014... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9016... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9018... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9020... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9022... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9024... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9026... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9028... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9030... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9032... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9034... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9036... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9038... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9040... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9042... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9044... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9046... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9048... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9050... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9052... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9054... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9056... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9058... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9060... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9062... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9064... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9066... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9068... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9070... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9072... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9074... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9076... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9078... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9080... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9082... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9084... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9086... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9088... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9090... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9092... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9094... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9096... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9098... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9100... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9102... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9104... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9106... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9108... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9110... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9112... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9114... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9116... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9118... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9120... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9122... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9124... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9126... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9128... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9130... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9132... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9134... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9136... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9138... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9140... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9142... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9144... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9146... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9148... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9150... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9152... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9154... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9156... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9158... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9160... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9162... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9164... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9166... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9168... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9170... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9172... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9174... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9176... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9178... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9180... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9182... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9184... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9186... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9188... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9190... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9192... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9194... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9196... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9198... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9200... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9202... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9204... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9206... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9208... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9210... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9212... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9214... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9216... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9218... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9220... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9222... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9224... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9226... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9228... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9230... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9232... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9234... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9236... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9238... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9240... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9242... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9244... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9246... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9248... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9250... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9252... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9254... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9256... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9258... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9260... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9262... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9264... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9266... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9268... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9270... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9272... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9274... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9276... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9278... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9280... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9282... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9284... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9286... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9288... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9290... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9292... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9294... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9296... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9298... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9300... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9302... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9304... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9306... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9308... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9310... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9312... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9314... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9316... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9318... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9320... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9322... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9324... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9326... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9328... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9330... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9332... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9334... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9336...
```

**Comment by sinegar** [comment]
In 3cx, the network config / setup have you used static or dynamic IP?

OCI, do you use a load balancer or have you put the instance in the public subnet (and have a public IP for the instance)?

**Comment by AFAskar** [comment]
a static ip
and i do have a public ip for the instance 
i don't know what you mean by a public subnet 
but it does say Public (Regional) in my subnet page of oci

**Comment by sinegar** [comment]
Few lines seems to be missing in the firewall output

Here is mine:
```
resolving 'stun-au.3cx.com'... done
resolving 'stun2.3cx.com'... done
resolving 'stun3.3cx.com'... done
resolving 'sip-alg-detector.3cx.com'... done
testing 3CX PhoneSystem 01 SIP Server... failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
stopping service... done
detecting SIP ALG... not detected
testing port 5060... Mapping does not match 5060. Mapping is 4916. ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
starting service... done
testing 3CX PhoneSystem Media Server... failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
stopping service... done
testing port 5090... Mapping does not match 5090. Mapping is 35429. ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing ports [9000..9398]... failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9000... Mapping does not match 9000. Mapping is 41822. ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
```

What is your result on port 5060? Do I see correctly that port 5090 passed the test?

—
Public subnet - OCI will create a virtual network for you and by default split it into into subnets (sub networks or ip ranges), a public (direct internet connectivity) and a private (no inbound internet access).

**Comment by AFAskar** [comment]
yes 5060 and 5090 have passed
but detecting SIP ALG... failed

**Comment by AFAskar** [comment]
here is another output of the test cause the above is incomplete
```
resolving 'stun-eu.3cx.com'... done
resolving 'stun2.3cx.com'... done
resolving 'stun3.3cx.com'... done
resolving 'sip-alg-detector.3cx.com'... done
testing 3CX PhoneSystem 01 SIP Server... failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
stopping service... done
detecting SIP ALG... failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 5060... done
starting service... done
testing 3CX PhoneSystem Media Server... failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
stopping service... done
testing port 5090... done
testing ports [9000..9398]... failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9000... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9002... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9004... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9006... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9008... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9010... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9012... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9014... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9016... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9018... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9020... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9022... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9024... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9026... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9028... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9030... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9032... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9034... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9036... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9038... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9040... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9042... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9044... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9046... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9048... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9050... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9052... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9054... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9056... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9058... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9060... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9062... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9064... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9066... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9068... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9070... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9072... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9074... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9076... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9078... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9080... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9082... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9084... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9086... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9088... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9090... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9092... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9094... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9096... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9098... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9100... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9102... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9104... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9106... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9108... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9110... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9112... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9114... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9116... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9118... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9120... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9122... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9124... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9126... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9128... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9130... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9132... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9134... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9136... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9138... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9140... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9142... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9144... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9146... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9148... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9150... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9152... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9154... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9156... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9158... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9160... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9162... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9164... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9166... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9168... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9170... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9172... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9174... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9176... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9178... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9180... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9182... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9184... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9186... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9188... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9190... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9192... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9194... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9196... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9198... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9200... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9202... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9204... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9206... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9208... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9210... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9212... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9214... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9216... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9218... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9220... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9222... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9224... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9226... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9228... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9230... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9232... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9234... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9236... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9238... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9240... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9242... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9244... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9246... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9248... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9250... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9252... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9254... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9256... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9258... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9260... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9262... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9264... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9266... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9268... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9270... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9272... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9274... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9276... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9278... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9280... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9282... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9284... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9286... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9288... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9290... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9292... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9294... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9296... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9298... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9300... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9302... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9304... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9306... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9308... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9310... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9312... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9314... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9316... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9318... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9320... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9322... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9324... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9326... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9328... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9330... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9332... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9334... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9336... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9338... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9340... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9342... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9344... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9346... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9348... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9350... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9352... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9354... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9356... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9358... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9360... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9362... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9364... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9366... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9368... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9370... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9372... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9374... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9376... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9378... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9380... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9382... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9384... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9386... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9388... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9390... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9392... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9394... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9396... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 9398... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing ports [10600..10998]... failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10600... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10602... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10604... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10606... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10608... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10610... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10612... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10614... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10616... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10618... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10620... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10622... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10624... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10626... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10628... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10630... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10632... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10634... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10636... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10638... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10640... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10642... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10644... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10646... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10648... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10650... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10652... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10654... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10656... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10658... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10660... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10662... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10664... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10666... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10668... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10670... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10672... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10674... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10676... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10678... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10680... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10682... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10684... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10686... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10688... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10690... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10692... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10694... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10696... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10698... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10700... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10702... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10704... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10706... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10708... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10710... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10712... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10714... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10716... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10718... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10720... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10722... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10724... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10726... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10728... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10730... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10732... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10734... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10736... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10738... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10740... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10742... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10744... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10746... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10748... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10750... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10752... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10754... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10756... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10758... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10760... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10762... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10764... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10766... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10768... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10770... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10772... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10774... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10776... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10778... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10780... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10782... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10784... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10786... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10788... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10790... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10792... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10794... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10796... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10798... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10800... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10802... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10804... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10806... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10808... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10810... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10812... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10814... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10816... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10818... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10820... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10822... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10824... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10826... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10828... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10830... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10832... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10834... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10836... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10838... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10840... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10842... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10844... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10846... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10848... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10850... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10852... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10854... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10856... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10858... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10860... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10862... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10864... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10866... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10868... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10870... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10872... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10874... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10876... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10878... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10880... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10882... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10884... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10886... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10888... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10890... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10892... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10894... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10896... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10898... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10900... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10902... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10904... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10906... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10908... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10910... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10912... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10914... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10916... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10918... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10920... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10922... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10924... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10926... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10928... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10930... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10932... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10934... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10936... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10938... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10940... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10942... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10944... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10946... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10948... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10950... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10952... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10954... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10956... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10958... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10960... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10962... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10964... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10966... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10968... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10970... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10972... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10974... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10976... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10978... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10980... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10982... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10984... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10986... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10988... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10990... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10992... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10994... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10996... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
testing port 10998... full cone test failed ([How to resolve?](http://www.3cx.com/blog/docs/firewall-checker))
starting service... done
```

**Comment by AFAskar** [comment]
all settings are in their default states 
FQDN is on local ip so off 
it's under a static ip

**Comment by AFAskar** [comment]
![image](https://user-images.githubusercontent.com/68832286/184460589-fdeb99e5-df19-4857-82be-b31596fbd2ed.png)
here is the screen shot of my security list

**Comment by sinegar** [comment]
That looks all good. Are you sure that the 9xxx ports are not blocked by the host firewall (ufw)? The 5060 and 5090 seems to pass.

**Comment by AFAskar** [comment]
i'm sure 
i did sudo ufw allow 7000:8999/udp
and 
sudo ufw allow 9000:10999/udp (v6)
and here is the ufw status output
![image](https://user-images.githubusercontent.com/68832286/184469301-784b30c3-62db-4a35-ad27-3fdcb2645d76.png)

**Comment by sinegar** [comment]
I checked my setup and I am actually not allowing any of the RTP ports. 

After the docs these are only required:

> Yes – if you intend on using remote extensions, WebRTC or a VoIP Provider

… _using the 3CX Tunnel Protocol / 3CX SBC_ (my assumption)

What client are you using?

**Comment by AFAskar** [comment]
the web client and the iPhone app

**Comment by sinegar** [comment]
The web client needs the WebRTC but the iPhone app should work fine. 

Have you tried to call *777 (echo test) from the iPhone app?

For the web client you could try host networking (`--network host`) instead of the port mappings ( `-p xxx` ).

**Comment by AFAskar** [comment]
Yes the echo test works i can hear myself 
And sorry for the late reply life got busy

**Comment by AFAskar** [comment]
And if i may ask does the echo test means that people using the app can talk to each other ?

**Comment by AFAskar** [comment]
Because i just use the webclient to test 
And also does this mean that the desktop client works as well or does it need webrtc

**Comment by AFAskar** [comment]
(`--network host`) instead of the port mappings ( `-p xxx` ).
 How would i do that is it in the docker run command?

**Comment by sinegar** [comment]
> Yes the echo test works i can hear myself
> And sorry for the late reply life got busy

no problem 

> And if i may ask does the echo test means that people using the app can talk to each other ?

My assumption is yes - as each client can communicate with the server

> Because i just use the webclient to test
And also does this mean that the desktop client works as well or does it need webrtc

again, my assumption is yes - based on https://www.3cx.com/docs/ports/

However, based on the ports page, some voip provider may use the UDP ports - mine does not. And in this case you may not be able place and receive external calls. It should be easy to test - just make a call. 

> (--network host) instead of the port mappings ( -p xxx ).
How would i do that is it in the docker run command?

Something like this (sorry didn’t test it):
```
docker run 
-d 
-t 
--tmpfs /tmp 
--tmpfs /run 
--tmpfs /run/lock 
-v /sys/fs/cgroup:/sys/fs/cgroup:ro 
--network host
ghcr.io/izer-xyz/3cx:latest
```

**Comment by AFAskar** [comment]
People using the app can’t call others just the echo test works and on the
app the connection is red

**Comment by sinegar** [comment]
Is the connection always red? Do you see the `ready for calls` message below the status?

That might be an issue with provisioning the client. Can you confirm the Network section of the Phone Provisioning of the users are correct?

(3CX admin dashboard / Users/ xxxx / Phone Provisioning / Network)

The `Network interface for registration and provisioning` should point to an IP or name that is accessible from the clients (by default it’s the docker container internal IP). And while you are there you could change the `SIP Transport` to TCP. 

(I am not sure where is the global setting so I set the network interface for each user)

**Comment by AFAskar** [comment]
I’m using my iphone and my web client

**Comment by flaviomeyer** [comment]
@sinegar I'm having the same issue as @abo3skr2019 described with Audio.

I'm using Oracle Cloud too with Oracle Linux 8 and firewall on OS disabled and all ports open on OCI to the machine.

My 3CX firewall has the same issues with 9000+ ports.

As SIP Provider I'm using Twilio 😊

Is there maybe the issue that the ports are not published on Docker side?

**Comment by sinegar** [comment]
Sorry, I didn’t have much time to test any of this. I have a simple config and not using the web client (which needs the UDP ports). 

To remove potential docker networking issues you could run with `host` networking:

```
docker run 
-d 
-t 
--tmpfs /tmp 
--tmpfs /run 
--tmpfs /run/lock 
-v /sys/fs/cgroup:/sys/fs/cgroup:ro 
--network host
ghcr.io/izer-xyz/3cx:latest
```

But there are way too many hops involved (OCI firewalls, host firewall, client connection config, …).

**Comment by flaviomeyer** [comment]
> Sorry, I didn’t have much time to test any of this. I have a simple config and not using the web client (which needs the UDP ports).
> 
> To remove potential docker networking issues you could run with `host` networking:
> 
> ```
> docker run 
> -d 
> -t 
> --tmpfs /tmp 
> --tmpfs /run 
> --tmpfs /run/lock 
> -v /sys/fs/cgroup:/sys/fs/cgroup:ro 
> --network host
> ghcr.io/izer-xyz/3cx:latest
> ```
> 
> But there are way too many hops involved (OCI firewalls, host firewall, client connection config, …).

Issue solved with `--network host`, thank you

**Comment by sinegar** [comment]
Closing all issues before archiving the project.
